### PR TITLE
Add new tool genomics-data-index

### PR DESCRIPTION
**Pull request by apetkau**
This adds a new tool, [genomics-data-index](https://github.com/apetkau/genomics-data-index), which can be used to index, query, cluster, and visualize genomics data across a range of features (nucleotide variants, kmers, genes).

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by apetkau** [comment]
@BiocondaBot please fetch artifacts

**Comment by apetkau** [comment]
@BiocondaBot please add label

File: recipes/genomics-data-index/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    - genomics_data_index
+  commands:
+    - gdi --version
+  requires:
+    - pip
+
+about:
+  home: https://github.com/apetkau/genomics-data-index
+  summary: Indexes genomics data (nucleotide variants, kmers, MLST) for fast querying of features.
+  license: APACHE
```
Comment (pvanheus) [inline recipes/genomics-data-index/meta.yaml:70]: Could you please use a SPDX identifier like [Apache-2.0](https://spdx.org/licenses/)?

**Review by pvanheus — CHANGES_REQUESTED** [review]

File: recipes/genomics-data-index/meta.yaml (hunk 1 above)
Comment (apetkau) [reply recipes/genomics-data-index/meta.yaml]: No problem. Done in dc35b9116cd452af1e7f324a3429b76533f6a09a. Thanks so much. I wasn't sure what the proper name should be.

**Review by pvanheus — APPROVED** [review]

**Comment by pvanheus** [comment]
@BiocondaBot please merge

**Comment by apetkau** [comment]
Thanks so much @pvanheus . I really appreciate it 👍

**Pull request merged**